.changesets/store-error-causes-as-transaction-metadata.md

```diff
@@ -0,0 +1,7 @@
+---
+bump: "patch"
```

Comment:
```suggestion
bump: "minor"
```

We're on patch version .14 now. This seems like a good new feature to bump the gem to a new minor version.